bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -115,14 +109,20 @@ spec:
           name: host-cni-netd
         - mountPath: /var/lib/cni/networks/openshift-sdn
           name: host-var-lib-cni-networks-openshift-sdn
+        - mountPath: /lib/modules
+          name: host-modules
```

Comment:
commit message doesn't explain what this part is about... was it intentional?

Reply:
It's needed for iptables; this fixes a bug that was copied over from openshift-ansible - namely, the volume was added but not the mount.

I'll update the commit